latch/registry/table.py

```diff
@@ -541,6 +560,119 @@ def _add_record_deletes_selection(
 
         mutations.append(res)
 
+    # upsert column
+
+    def create_column(
+        self,
+        key: str,
+        type: Union[Type[RegistryPrimitivePythonValue], str, List[str]],
```

Comment:
This is bad -- there should be explicit classes which indicate the types

Union[Type[RegistryPrimitivePythonValue], RegistryPrimitiveLinkType, RegistryPrimitiveEnumType]